docs/about/technical_steering_committee.md

```diff
@@ -1,55 +1,126 @@
 
 On this page we describe:
 
+- [Kedro TSC roles](#kedro-tsc-roles)
+- [Responsibilities of a voting member](#responsibilities-of-a-voting-member)
+- [Process to become a voting member](#process-to-become-a-voting-member)
 - [Responsibilities of a maintainer](#responsibilities-of-a-maintainer)
-- [Requirements to become a maintainer](#requirements-to-become-a-maintainer)
-- [Current maintainers](#current-maintainers)
-- [Past maintainers](#past-maintainers)
-- [Application process](#application-process)
+- [Process to become a maintainer](#process-to-become-a-maintainer)
+- [Responsibilities of an advisor](#responsibilities-of-an-advisor)
+- [Process to become an advisor](#process-to-become-an-advisor)
+- [Current Kedro TSC members](#current-kedro-tsc-members)
+- [Past Kedro TSC members & maintainers](#past-kedro-tsc-members-and-maintainers)
 - [Voting process](#voting-process)
+- [Adding or removing TSC members](#adding-or-removing-tsc-members)
 
-## Responsibilities of a maintainer
+## Kedro TSC roles
+
+The Kedro TSC is responsible for the project's future development. The TSC is made up of the following roles:
+
+### Voting members
+
+Voting Members form the core of the TSC and hold the right to vote on both code modifications and membership additions. They are expected to make significant contributions to the strategy, direction, and governance of the project. Their primary responsibility is to ensure the long-term success of Kedro.
+
+### Maintainers
+
+Maintainers have repository access and full participation rights in discussions but do not hold voting privileges. They are active contributors who focus on different areas of the project. Maintainers are responsible for maintaining the health of the project, participating in planning and (technical) design decisions, and otherwise acting as part of the core Kedro development team.
+
+### Advisors
+
+Advisors are individuals who are not actively contributing to the codebase or attending regular meetings but provide valuable expertise, feedback, and strategic guidance to the Kedro project.
+
+## Responsibilities of a voting member
+
+- Defining and maintaining the product vision and roadmap
+- Supporting key technical and strategic initiatives
+- Providing input on complex or high-impact technical decisions
+- Advising and mentoring contributors
+- Representing Kedro within the broader community
+- Attend regular team meetings to discuss the project plans and roadmap
+
+### Voting rights
+Voting members take part in two types of votes:
+
+- Code modifications – follows the [KEP process](https://github.com/kedro-org/kedro/discussions/5150)
+- Membership additions – new voting members must receive majority approval (≥50%) from existing voting members (see more details below)
+
+
+## Process to become a voting member
+Just contributing does not make you a voting member. You need to show commitment to Kedro's long-term success by working with existing TSC members for at least one year.
```

Comment:
I think they should be promoted from one of the existing non-voting roles? As in, it doesn't make sense to me to become a voting member directly, but rather to have been part of the TSC for at least one year prior (not just working with them).

Reply:
I agree with @deepyaman , internal org structures for McKinsey shouldn't dictate how an open project operate. Probably 12 months is too long though, as I imagine that six months should be enough for someone to be committed for the long-term success of the project.

Reply:
Thanks all! After another quick discussion we feel this makes sense. I'll update the text accordingly.